pkg/controllers/user/monitoring/metricExpression.go

```diff
@@ -2732,8 +2732,7 @@ metadata:
     metric: disk-wal-fsync-duration-seconds-sum-quantile
     source: rancher-monitoring
 spec:
-  expression: sum(histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])))
-    by (instance)
+  expression: histogram_quantile(0.99, sum(rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])) by (instance, le))
   legendFormat: WAL fsync([[instance]])
```

Comment:
same here.

Reply:
This is a good expression. For one instance, `histogram_quantile` should be the outside function.